README.md

```diff
@@ -68,9 +68,9 @@ At most, 5 kube-state-metrics and 5 [kubernetes releases](https://github.com/kub
 
 | kube-state-metrics | **Kubernetes 1.17** |  **Kubernetes 1.18** |  **Kubernetes 1.19** |  **Kubernetes 1.20** |  **Kubernetes 1.21** |
 |--------------------|---------------------|----------------------|----------------------|----------------------|----------------------|
-| **v1.8.0**         |         -           |          -           |          -           |          -           |          -           |
 | **v1.9.8**         |         -           |          -           |          -           |          -           |          -           |
 | **v2.0.0**         |         -/✓         |          -/✓         |          ✓           |          ✓           |          -/✓         |
+| **v2.1.0-rc.0**    |         -/✓         |          -/✓         |          ✓           |          ✓           |          ✓           |
```

Comment:
I don't think we need an rc, we can go straight to the release of v2.1, we mainly did this for 2.0 due to so many breaking changes. What do you think?

Reply:
This works for me as well, I did it as the release process suggested to start with an rc. I don't mind going straight to 2.1.0 and branch off then. the risk should be rather low.

Reply:
I think we should still follow the documented release process.

But having said that, if we want to retire the rc builds, I am okay with amending the release process in another PR and going forward with this approach